.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
not sure why you would ever commit a personal ide extension to a library, this has no place here

Reply:
Purely for the debugging workflow